data/virt_autotest/virt_logs_collector.sh

```diff
@@ -344,6 +344,51 @@ function collect_extra_logs_from_host() {
 	return ${ret_result}
 }
 
+#This function supports collecting guest configure xml file.
+#Typical usage: collect_guest_config logs_folder guest
+function collect_guest_config() {
+        local logs_folder=$1
+        local guest_name=$2
+        local guest_config="${logs_folder}/${guest_name}"
+
+        #Collect guest configure xml file
+        echo -e "Collect ${guest_name} configure xml file"
+        virsh dumpxml ${guest_name} | tee ${guest_config}-config.xml
+        if [[ $? -eq 0 ]];then
+           return 0
+        else
+           return 1
+        fi
+}
+
+#This function supports collecting and validating guest autoyast file.
+#Typical usage: collect_guest_autoyast logs_folder guest
+function collect_guest_autoyast() {
+        local logs_folder=$1
+        local guest_name=$2
+        local autoyast_dir="/srv/www/htdocs/install/autoyast"
+        local guest_autoyast="${autoyast_dir}/${guest_name}"
+        local guest_profile="/usr/share/YaST2/schema/autoyast/rng/profile.rng"
+
+        #Install required packages
+        zypper_cmd="zypper --non-interactive in libxml2-tools yast2-schema"
```

Comment:
Suggest to use zypper_call instead.

Reply:
this is virt_logs_collector.sh shell script, should be not call zypper_call() here, imho.

Reply:
you are right, sorry for not noticing it is inside shell.